package/MDAnalysis/coordinates/GRO.py

```diff
@@ -239,19 +240,20 @@ class GROWriter(base.WriterBase):
     }
     fmt['xyz_v'] = fmt['xyz'][:-1] + "{vel[0]:8.4f}{vel[1]:8.4f}{vel[2]:8.4f}\n"
 
-    def __init__(self, filename, convert_units=None,  **kwargs):
+    def __init__(self, filename, convert_units=None, n_atoms=None):
```

Comment:
You need to keep the **kwargs sinkhole argument though to support our API which allows different keyword args for different formats